ipalib/cli.py

```diff
@@ -49,7 +49,7 @@
 from six.moves import input
 
 from ipalib.util import (
-    check_client_configuration, get_terminal_height, open_in_pager
+    check_client_configuration, get_terminal_height, open_in_pager, get_pager
```

Comment:
Really minor and insignificant style nit pick: try to keep imports in alphanumeric order.